FizzBuzz/spec/fizzbuzz_spec.rb

```diff
@@ -0,0 +1,44 @@
+require "spec_helper"
+require 'FizzBuzzImproved'
+
+describe FizzBuzzImproved do
+  it "returns FizzBuzzSivv if input is 105" do
+    # Given an input of 105
+    # When 3 returns "Fizz", 5 returns "Buzz", and 7 returns "Sivv"
+    # Then return should be "FizzBuzzSivv"
+
+    a = FizzBuzzImproved.new
```

Comment:
you could add this to a before(:each) do block so you don't have to add this line in every test.